Update for latest ADAM snapshot changes

```
[ERROR] <cli/src/main/scala/org/bdgenomics/cannoli/cli/Cannoli.scala>:132: error: overriding method configure in class AbstractModule of type ()Unit;
[ERROR]  method configure needs `override' modifier
[ERROR]   def configure() {
[ERROR]       ^
[WARNING] <cli/src/main/scala/org/bdgenomics/cannoli/cli/InterleaveFastq.scala>:108: warning: constructor AsciiLineReader in class AsciiLineReader is deprecated: see corresponding Javadoc for more information.
[WARNING]     val pair1Reader = new AsciiLineReaderIterator(new AsciiLineReader(cIs1))
[WARNING]                                                   ^
[WARNING] <cli/src/main/scala/org/bdgenomics/cannoli/cli/InterleaveFastq.scala>:109: warning: constructor AsciiLineReader in class AsciiLineReader is deprecated: see corresponding Javadoc for more information.
[WARNING]     val pair2Reader = new AsciiLineReaderIterator(new AsciiLineReader(cIs2))
[WARNING]                                                   ^
[WARNING] two warnings found
[ERROR] one error found
```

--- a/cli/src/main/scala/org/bdgenomics/cannoli/cli/Cannoli.scala
+++ b/cli/src/main/scala/org/bdgenomics/cannoli/cli/Cannoli.scala
@@ -129,7 +129,7 @@ class Cannoli @Inject() (commandGroups: List[CommandGroup]) extends Logging {
  * Cannoli module, binds the default list of command groups.
  */
 class CannoliModule extends AbstractModule with ScalaModule {
-  def configure() {
+  override def configure() {
     bind[List[CommandGroup]].toInstance(Cannoli.defaultCommandGroups)
   }
 }

--- a/cli/src/test/scala/org/bdgenomics/cannoli/cli/CannoliSuite.scala
+++ b/cli/src/test/scala/org/bdgenomics/cannoli/cli/CannoliSuite.scala
@@ -97,7 +97,7 @@ class CannoliSuite extends FunSuite {
     val stream = new ByteArrayOutputStream()
     Console.withOut(stream) {
       val module = new AbstractModule with ScalaModule {
-        def configure() = {
+        override def configure() = {
           bind[List[CommandGroup]].toInstance(List(CommandGroup("SINGLE COMMAND GROUP", List(Freebayes))))
         }
       }
@@ -115,7 +115,7 @@ class CannoliSuite extends FunSuite {
     val stream = new ByteArrayOutputStream()
     Console.withOut(stream) {
       val module = new AbstractModule with ScalaModule {
-        def configure() = {
+        override def configure() = {
           bind[List[CommandGroup]].toInstance(defaultCommandGroups.union(List(CommandGroup("NEW COMMAND GROUP", List(Freebayes)))))
         }
       }

--- a/core/src/main/scala/org/bdgenomics/cannoli/util/QuerynameGroupingIterator.scala
+++ b/core/src/main/scala/org/bdgenomics/cannoli/util/QuerynameGroupingIterator.scala
@@ -38,7 +38,7 @@ private[util] class QuerynameGroupingIterator(
 
     val readsInFragment = ListBuffer.empty[AlignmentRecord]
     val readName = bufferedIter.head.getReadName
-    val optFragmentSize = Option(bufferedIter.head.getInferredInsertSize)
+    val optInsertSize = Option(bufferedIter.head.getInferredInsertSize)
       .map(_.toInt)
 
     @tailrec def fillList() {
@@ -54,11 +54,11 @@ private[util] class QuerynameGroupingIterator(
     fillList()
 
     val fb = Fragment.newBuilder
-      .setReadName(readName)
+      .setName(readName)
       .setAlignments(readsInFragment)
 
     // set the insert size if known
-    optFragmentSize.foreach(fs => fb.setFragmentSize(fs))
+    optInsertSize.foreach(fs => fb.setInsertSize(fs))
 
     fb.build
   }

--- a/core/src/test/scala/org/bdgenomics/cannoli/util/QuerynameGroupingIteratorSuite.scala
+++ b/core/src/test/scala/org/bdgenomics/cannoli/util/QuerynameGroupingIteratorSuite.scala
@@ -73,8 +73,8 @@ class QuerynameGroupingIteratorSuite extends FunSuite {
     assert(readNames.count(_ == "read1") === 1)
     assert(readNames.count(_ == "read2") === 2)
     assert(readNames.count(_ == "read3") === 3)
-    val read2Opt = fragments.find(_.getReadName == "read2")
+    val read2Opt = fragments.find(_.getName == "read2")
     assert(read2Opt.isDefined)
-    read2Opt.foreach(r => assert(r.getFragmentSize === 100))
+    read2Opt.foreach(r => assert(r.getInsertSize === 100))
   }
 }

--- a/pom.xml
+++ b/pom.xml
@@ -342,7 +342,12 @@
   </build>
   <dependencyManagement>
     <dependencies>
-      <dependency>
+     <dependency>
+        <groupId>com.google.guava</groupId>
+        <artifactId>guava</artifactId>
+        <version>27.0-jre</version>
+     </dependency>
+     <dependency>
         <groupId>junit</groupId>
         <artifactId>junit</artifactId>
         <version>${junit.version}</version>

